subsys/bluetooth/host_extensions/host_extensions.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/*
+ * Purpose of this file is to add nordic-LL only vendor-specific commands.
+ * Adding them in nrf is better maintainable.
+ */
+
+#if defined(CONFIG_BT_TRANSMIT_POWER_CONTROL)
+
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/hci.h>
+#include <../subsys/bluetooth/host/conn_internal.h>
+
+#include "hci_types_host_extensions.h"
+#include <bluetooth/nrf/host_extensions.h>
+
+/* Write Remote Transmit Power Level HCI command */
+int bt_conn_set_remote_tx_power_level(struct bt_conn *conn,
+						enum bt_conn_le_tx_power_phy phy, int8_t delta)
```

Comment:
Nit: some weird formatting here (at least as displayed by github). Maybe run clang-format on all the new files?